### PR TITLE
Increase default number of retries for IPC connections

### PR DESCRIPTION
**Pull request by almiliMSFT**

### CODE REVIEW

File: Public/Src/Utilities/Ipc/Common/Utils.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ internal static async Task<Possible<TConn>> ConnectAsync<TConn>(int maxRetry, Ti
                 if (numAttempts > 0)
                 {
                     // TODO: consider implementing exponential backoff instead
```
Comment (mpysson) [inline Public/Src/Utilities/Ipc/Common/Utils.cs]: You added a linear backoff. Is that enough to warrant removing the todo?

Comment (almiliMSFT) [reply Public/Src/Utilities/Ipc/Common/Utils.cs]: Probably.  I'll remove it.

**Pull request merged**